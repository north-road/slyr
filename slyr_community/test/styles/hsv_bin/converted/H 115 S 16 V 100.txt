[{'class': 'QgsSimpleFillSymbolLayer',
  'color': '214,255,214,255,rgb:0.8392157,1,0.8392157,1',
  'joinstyle': 'round',
  'outline_color': '255,0,0,255,rgb:1,0,0,1',
  'outline_width': '8',
  'outline_width_unit': 'Point'}]
