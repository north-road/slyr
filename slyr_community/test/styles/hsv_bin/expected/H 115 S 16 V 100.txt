{'levels': [{'color': {'B': 214,
                       'G': 255,
                       'R': 214,
                       'dither': False,
                       'is_null': False,
                       'stream_offset': '0xb4',
                       'type': 'HsvColor',
                       'version': 1},
             'enabled': True,
             'fill_style': 'solid',
             'locked': False,
             'outline': {'color': {'B': 0,
                                   'G': 0,
                                   'R': 255,
                                   'dither': True,
                                   'is_null': False,
                                   'stream_offset': '0x71',
                                   'type': 'RgbColor',
                                   'version': 1},
                         'enabled': True,
                         'line_type': 'solid',
                         'locked': False,
                         'stream_offset': '0x5f',
                         'symbol_level': 0,
                         'type': 'SimpleLineSymbol',
                         'version': 1,
                         'width': 8.0},
             'stream_offset': '0x4d',
             'symbol_level': 0,
             'type': 'SimpleFillSymbol',
             'version': 1}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerFillSymbol',
 'version': 2}
